config/config.go

```diff
@@ -35,6 +35,7 @@ type Global struct {
 	DialMode                  string        `mapstructure:"dial_mode" default:"domain"`
 	DisableWaitingNetwork     bool          `mapstructure:"disable_waiting_network" default:"false"`
 	AutoConfigKernelParameter bool          `mapstructure:"auto_config_kernel_parameter" default:"false"`
+	AutoConfigFirewallRule    bool          `mapstructure:"auto_config_firewall_rule" default:"false"`
```

Comment:
Not sure if it's deprecated or "a placeholder for future use". cc @mzz2017

Reply:
1. A better syntax should be added to tell dae-wing this is deprecated and shouldn't be generated. This can be done in another work.
1. Add a `DEPRECATED:` comment for this variable is recommended by go team.

Reply:
Added comment above `AutoConfigFirewallRule`: https://github.com/daeuniverse/dae/pull/458/commits/7c924d193d6ca5852abc0850c4d61a79519f2765